Web-Editor.md

```diff
@@ -8,6 +8,8 @@ ___
 
 Editor sessions are first created by executing a command on the server.
 
+You can run different commands to open sessions with different scopes:
```

Comment:
I would suggest to have it as a new line and not a new paragraph.

So
```markdown
Editor sessions are first created by executing a command on the server.  
You can run different commands to open sessions with different scopes:
```

Instead of what you have now